DPG/Tasks/AOTTrack/PID/qaPIDTPC.cxx

```diff
@@ -76,14 +79,17 @@ struct tpcPidQa {
   static constexpr std::string_view hsignal_wTOF[Np] = {"wTOF/signal/El", "wTOF/signal/Mu", "wTOF/signal/Pi",
                                                         "wTOF/signal/Ka", "wTOF/signal/Pr", "wTOF/signal/De",
                                                         "wTOF/signal/Tr", "wTOF/signal/He", "wTOF/signal/Al"};
+  static constexpr std::string_view hnsigma_eta_wTOF[Np] = {"wTOF/nsigma/eta/El", "wTOF/nsigma/eta/Mu", "wTOF/nsigma/eta/Pi",
+                                                            "wTOF/nsigma/eta/Ka", "wTOF/nsigma/eta/Pr", "wTOF/nsigma/eta/De",
+                                                            "wTOF/nsigma/eta/Tr", "wTOF/nsigma/eta/He", "wTOF/nsigma/eta/Al"};
 
   HistogramRegistry histos{"Histos", {}, OutputObjHandlingPolicy::AnalysisObject};
 
   Configurable<int> logAxis{"logAxis", 1, "Flag to use a log momentum axis"};
   Configurable<int> nBinsP{"nBinsP", 3000, "Number of bins for the momentum"};
   Configurable<float> minP{"minP", 0.01, "Minimum momentum in range"};
   Configurable<float> maxP{"maxP", 20, "Maximum momentum in range"};
-  ConfigurableAxis etaBins{"etaBins", {100, -1.f, 1.f}, "Binning in eta"};
+  ConfigurableAxis etaBins{"etaBins", {50, -1.f, 1.f}, "Binning in eta"};
```

Comment:
I think leave the default eta binning as it was, just we make sure to change the binning in the config when we run including the THnSparse